doc/source/how-to/documenting.rst

```diff
@@ -469,349 +470,210 @@ branch within this repository. This is done by uploading the generated
+your documentation for the first time.
+
+.. warning::
+
+    Deploying your documentation to another repository is discouraged. It
+    translates to more maintenance work and does not support the multi-version
+    documentation.
+
+For deploying the documentation to another repository, use the following workflow:
 
-.. admonition:: Organization approval to use PyAnsys bot
+.. code-block:: yaml
 
-    You must have internal approval to use the PyAnsys bot because your repository must
-    be added to its list of repositories. For more information, email
-    `PyAnsys Support <pyansys.support@ansys.com>`_.
+    doc-deploy:
+      name: "Deploy documentation to a different repo"
+      runs-on: ubuntu-latest
+      needs: doc-build
+      steps:
+        - name: "Deploy documentation"
+          uses: pyansys/actions/doc-deploy-to-repo@v1
+          with:
+            cname: "<library>.docs.pyansys.com"
+            repository: "<owner>/<repository-name>"
+            bot-id: ${{ secrets.BOT_APPLICATION_ID }}
+            bot-token: ${{ secrets.BOT_APPLICATION_PRIVATE_KEY }}
 
 
-Once your repository has been added to the white-list for the bot repositories, you must
-add the following code to your CICD YAML file for the authentication:
+Supporting multi-version documentation
+--------------------------------------
+With the release of `pyansys/actions@v1
+<https://actions.docs.pyansys.com/release/0.1/index.html>`_, projects can
+benefit from multi-version documentation. Projects taking advantage of this
+feature need to apply different configurations according to their level of
+maturity.
 
-.. tab-set::
+Common configuration
+~~~~~~~~~~~~~~~~~~~~
+Follow these steps to use the multi-version documentation:
 
-    .. tab-item:: Using ``tox``
+- Create a ``gh-pages`` branch in the repository of your project.
+- In this branch, include a ``CNAME`` containing the canonical name for the webpage hosting the documentation.
+- Create also a ``.nojekyll`` file to indicate GitHub about the nature of the website.
+- Create a ``release/`` directory and a ``versions.json`` file inside of it.
 
-        .. code-block:: yaml
+The content of the ``version.json`` file should read as follows:
 
-          - name: Get Bot Application Token
-            if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-            id: get_workflow_token
-            uses: peter-murray/workflow-application-token-action@v1
-            with:
-              application_id: ${{ secrets.BOT_APPLICATION_ID }}
-              application_private_key: ${{ secrets.BOT_APPLICATION_PRIVATE_KEY }}
+.. code-block:: json
 
+    [
+      {
+        "version": "dev",
+        "url": "<cname>"
+      }
+    ]
 
-    .. tab-item:: Without using ``tox``
 
-        .. code-block:: yaml
+- Update the version for the ``ansys-sphinx-theme`` to support ``>=0.7``.
+- Include the following lines in :ref:`The \`\`conf.py\`\` file`:
 
```

Comment:
I'm not the best at reading between deleted and added lines. However, I don't see new lines to replace lines 680-686. I only see the tagging for a code block. Does "import os" to be moved prior to the "Additionally, you must...."?